ui/webui/src/index.js

```diff
@@ -31,6 +31,7 @@ import { Application } from "./components/app.jsx";
  */
 import "./lib/patternfly/patternfly-4-overrides.scss";
 import "./components/app.scss";
+import "./components/animations-workaround.css";
```

Comment:
This is for the progress component, right? 
Should be added in src/components/installation/InstallationProgress.scss or at leat in src/components/installation/ folder.